recipes/julia-mode

```diff
@@ -1,4 +1,4 @@
 (julia-mode
- :repo "JuliaLang/julia"
+ :repo "JuliaLang/julia-emacs"
  :fetcher github
- :files ("contrib/julia-mode.el"))
+ :files ("julia-mode.el"))
```

Comment:
This line is unnecessary now I think. The test file will be ignore, and there are no other .el files in the repo.